noir/aztec_macros/src/lib.rs

```diff
@@ -31,6 +34,16 @@ impl MacroProcessor for AztecMacro {
         transform(ast, crate_id, context)
     }
 
+    fn process_unresolved_traits_impls(
+        &self,
+        crate_id: &CrateId,
+        context: &mut HirContext,
+        unresolved_traits_impls: &Vec<UnresolvedTraitImpl>,
+        collected_functions: &mut Vec<UnresolvedFunctions>,
+    ) -> Result<(), (MacroError, FileId)> {
+        inject_compute_note_hash_and_nullifier(crate_id, context, unresolved_traits_impls, collected_functions)
```

Comment:
You need to check that we have an `aztec` dependency as well. This will fail Noir CI for contract tests that do not depend on `aztec`.